### ISSUE TITLE
[BUG] kirby-x-banner-image content not properly shown on desktop

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#3590]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue)

## Describe the bug
If I package the `@kirbydesign/extensions-angular` and install it in a fresh Angular 17 project, 

## Describe how to reproduce the bug

1. From the root of our repo, run `npx nx build extensions-angular && npm pack libs/extensions/angular/dist`
2. In a fresh Angular 17 project, install the produced tgz file, e.g. add the tgz file to the root of the fresh application and add `"@kirbydesign/extensions-angular": "file:kirbydesign-extensions-angular-0.0.1.tgz",` to dependencies in `package.json`
3. Import `ImageBannerComponent` and add it to your template
```
<kirby-x-image-banner
  [title]="'An Image Banner'"
  [imagePath]="'https://images.unsplash.com/photo-1514415008039-efa173293080'"
  [bodyText]="'This is the body text.'"
  [actionButtonText]="'Read more'"
  [backgroundBlur]="'dark'"
></kirby-x-image-banner>
``` 
4. Content inside the banner is clipped when viewport width is above the `small` breakpoint (632px):
<img width="645" alt="image" src="https://github.com/user-attachments/assets/282ef9c0-b19a-4f12-945a-291edbaf0098">

## Which Kirby version was used?
Unreleased extensions-angular library.

## What was the expected behavior?
I would expect all content to show like this:
<img width="715" alt="image" src="https://github.com/user-attachments/assets/d1c8741a-3830-4d6a-a6f0-f5ee48a29e61">

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
The issue can be "fixed" by applying something like `height: 100%` to `.content-layer` inside kirby-card, as its height is currently 0:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/a9b65e73-d304-4e9d-8a5a-daf4418ffac5">

So the content layer is collapsing as the height of the content inside is 0, and we should probably fix this by letting the height of the banner be defined by the elements inside the nested `kirby-card`, not by setting a fixed height directly on the `kirby-card` element.

**Comment by RasmusKjeldgaard** [comment]
Turns out the 100% height fix was applied in #3492, that was released in v9.6.0. This bug was identified with Kirby v9.5.0, and v9.6.0 fixes the issue.
